common/achievement/get.ts

```diff
@@ -73,7 +123,9 @@ const assembleAchievementData = async (userAchievements: User_achievement[], use
             condition,
             currentAchievementTemplate.conditionDataAggregations as ConditionDataAggregations,
             currentAchievementTemplate.metrics,
-            userAchievements[currentAchievementIndex].recordValue
+            userAchievements[currentAchievementIndex].recordValue,
+            user.userID,
+            userAchievements[currentAchievementIndex].context
         );
         currentValue = dataAggregationKeys.map((key) => evaluationResult.resultObject[key]).reduce((a, b) => a + b, 0);
```

Comment:
I think the `currentValue` doesn't work right here for sequential achievements, because we always check the current template
<img width="400" alt="Bildschirmfoto 2023-12-18 um 08 42 55" src="https://github.com/corona-school/backend/assets/115979086/4ecf1985-cffe-4c9d-9a8b-a27c6168a40e">

Reply:
The issue was found and fixed!